src/DemoBundle/Resources/config/routing.yml

```diff
@@ -1,3 +1,5 @@
 post_api:
-    type: sylius.api
-    resource: acme.post
+    type: sylius.resource_api
```

Comment:
I based my opinion on Sylius configuration. Check [admin api routing](https://github.com/Sylius/Sylius/tree/master/src/Sylius/Bundle/AdminApiBundle/Resources/config/routing). For example [zones](https://github.com/Sylius/Sylius/blob/master/src/Sylius/Bundle/AdminApiBundle/Resources/config/routing/zone.yml), [taxons](https://github.com/Sylius/Sylius/blob/master/src/Sylius/Bundle/AdminApiBundle/Resources/config/routing/taxon.yml), [tax rates](https://github.com/Sylius/Sylius/blob/master/src/Sylius/Bundle/AdminApiBundle/Resources/config/routing/tax_rate.yml) or [tax categories](https://github.com/Sylius/Sylius/blob/master/src/Sylius/Bundle/AdminApiBundle/Resources/config/routing/tax_category.yml)